slackify-html.js

```diff
@@ -10,30 +10,37 @@ module.exports = function slackify(html) {
   var parser = new htmlparser.Parser(handler);
   parser.parseComplete(html);
   var dom = handler.dom;
-  if (dom)
-    return entities.decode(walk(dom));
-  else
-    return '';
+  if (dom) return entities.decode(walk(dom));
+  else return "";
 };
 
 function walkList(dom, ordered, nesting, start) {
```

Comment:
this function here is the only thing that changed you can ignore all the changes to the rest of the file